packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -107,6 +110,91 @@ export class FromCloudFormation {
     return ret;
   }
 
+  public static parseCreationPolicy(policy: any): CfnCreationPolicy | undefined {
+    if (policy == null) { return undefined; }
+
+    // change simple JS values to their CDK equivalents
+    policy = FromCloudFormation.parseValue(policy);
+
+    return {
+      autoScalingCreationPolicy: parseAutoScalingCreationPolicy(policy.AutoScalingCreationPolicy),
+      resourceSignal: parseResourceSignal(policy.ResourceSignal),
+    };
```

Comment:
Should we return `undefined` if this struct is empty. 

The current logic returns `{}` when `policy` is not an actual creation policy. Is this how we want it?

Reply:
Sure.